corehq/apps/commtrack/sms.py

```diff
@@ -131,6 +131,7 @@ def parse(self, text):
             RequisitionActions.FULFILL,
             RequisitionActions.RECEIPTS
         ]:
+            # todo: soft assert? remove?
```

Comment:
yeah i think this can all be removed (maybe we can add soft asserts along with the other sms functionality you're also removing)